tests/garage/torch/algos/test_vpg.py

```diff
@@ -0,0 +1,91 @@
+import gym
+import pytest
+import torch
+
+from garage.envs.base import GarageEnv
+from garage.experiment import deterministic
+from garage.experiment import LocalRunner
+from garage.np.baselines import LinearFeatureBaseline
+from garage.torch.algos import VPG
+from garage.torch.policies import GaussianMLPPolicy
+from tests.fixtures import snapshot_config
+
+# yapf: disable
```

Comment:
i think this disables yapf for everything below this line. please only disable yapf for blocks which need it.

Reply:
add `#yapf: enable` on the bottom